CHANGELOG.md

```diff
@@ -2,11 +2,22 @@
 
 ## Unreleased
 
+### Features
+
+- Add support for 16KB page sizes (Android 15) ([#3620](https://github.com/getsentry/sentry-java/pull/3620))
+
 ### Fixes
 
 - Load lazy fields on init in the background ([#3803](https://github.com/getsentry/sentry-java/pull/3803))
 - Replace setOf with HashSet.add ([#3801](https://github.com/getsentry/sentry-java/pull/3801))
 
+### Dependencies
+
+- Bump Native SDK from v0.7.2 to v0.7.8 ([#3620](https://github.com/getsentry/sentry-java/pull/3620))
```

Comment:
native 0.7.4+ [needed to be in v8](https://github.com/getsentry/sentry-java/pull/3441#pullrequestreview-2076732637)
Or did anything change?

Reply:
As discussed, it's actually fine to bump `sentry-native`, as the NDK move does still allow us to directly include sentry-native in `7.x`